File: CS/09_Charts/RangeOfChartData.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.IO;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Charts;

namespace RangeOfChartData
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PPT document
            Presentation ppt = new Presentation();

            //Load PPT file 
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\ChartSample2.pptx");

            //Create a StringBuilder object
            StringBuilder sb = new StringBuilder();

            //Get chart on the first slide
            IChart chart = ppt.Slides[0].Shapes[0] as IChart;
            if (chart != null)
            {
                int lastRow = chart.ChartData.LastRowIndex;
                int lastCol = chart.ChartData.LastColIndex;
                sb.AppendLine("lastRowIndex: " + lastRow + "\r\n" + "lastColIndex: " + lastCol);
            }

            //Save to txt file
            String result = "output.txt";
            File.WriteAllText(result, sb.ToString());

            System.Diagnostics.Process.Start(result);
        }
    }
}

File: VB/09_Charts/RangeOfChartData.vb
Imports System.ComponentModel
Imports System.IO
Imports System.Text
Imports Spire.Presentation
Imports Spire.Presentation.Charts

Namespace RangeOfChartData
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim ppt As New Presentation()

			'Load PPT file 
			ppt.LoadFromFile("..\..\..\..\..\..\Data\ChartSample2.pptx")

			'Create a StringBuilder object
			Dim sb As New StringBuilder()

			'Get chart on the first slide
			Dim chart As IChart = TryCast(ppt.Slides(0).Shapes(0), IChart)
			If chart IsNot Nothing Then
				Dim lastRow As Integer = chart.ChartData.LastRowIndex
				Dim lastCol As Integer = chart.ChartData.LastColIndex
				sb.AppendLine("lastRowIndex: " & lastRow & vbCrLf & "lastColIndex: " & lastCol)
			End If

			'Save to txt file
			Dim result As String = "output.txt"
			File.WriteAllText(result, sb.ToString())

			Process.Start(result)
		End Sub
	End Class
End Namespace